sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -580,7 +579,8 @@ def _download_blob_options(self, offset=None, length=None, validate_content=Fals
                                encryption_algorithm=cpk.algorithm)
 
         options = {
-            'service': self._client.blob,
+            'client': self._client.blob,
+            'clients': self._client,
```

Comment:
what's the significance of having both?

Reply:
I planned to remove client if you are fine with having separating download.py for file and blob. I commented in download_async.py. ^_^